hrvisa/tests/phpunit/CRM/HRVisa/ActivityTest.php

```diff
@@ -56,10 +37,12 @@ function setUp() {
     $this->createLoggedInUser();
   }
 
+  function tearDown()
+  {
+
+  }
+
   protected static function _populateDB($perClass = FALSE, &$object = NULL) {
-    if (!parent::_populateDB($perClass, $object)) {
-      return FALSE;
-    }
     _hrvisa_phpunit_populateDB();
```

Comment:
weird ... hrvisa one of the view extensions that its tests run without failing .. let me check that again .